src/main/java/com/adobe/commerce/cif/model/cart/Cart.java

```diff
@@ -35,20 +35,20 @@
     @ApiModelProperty(value = "If set, this defines the customer owning this cart. If not set, the cart is an anonymous cart.")
     protected String customerId;
     
-    @ApiModelProperty(value = "The net total price for the cart, including discounts, and shipping, but excluding any taxes.")
+    @ApiModelProperty(value = "The net total price for the cart, including discounts, and shipping, but excluding any taxes.", required = true)
     protected Price netTotalPrice;
     
-    @ApiModelProperty(value = "The gross total price for the cart, including discounts, shipping, and all taxes.")
+    @ApiModelProperty(value = "The gross total price for the cart, including discounts, shipping, and all taxes.", required = true)
     protected Price grossTotalPrice;
```

Comment:
This should not be required, it can only be set once the tax model is known, so sometimes only after the shipping address is set.